### PR TITLE
Fix some styling

### PR DESCRIPTION
**Pull request by balloob**
- Show placeholder when loading create-link
- Remove accidental h1 active on placeholder content
- Update instance info styling on index
- Move instance info out of the card on redirect page
- Move footer to the bottom

image

image

### CODE REVIEW

File: src/entrypoints/my-redirect.ts [hunk 1]
```diff
@@ -1,112 +1,95 @@
...
-      return true;
-    }
-    if (type === "url") {
-      return value && value === sanitizeUrl(value);
-    }
-    return false;
+  let changeInstanceEl = document.querySelector(".instance-footer");
+  if (!changeInstanceEl) {
+    changeInstanceEl = document.createElement("div");
+    changeInstanceEl.classList.add("instance-footer");
```
Comment (bramkragten) [inline src/entrypoints/my-redirect.ts:77]: Why don't we add this to `redirect-page.html`?

Comment (balloob) [reply src/entrypoints/my-redirect.ts:77]: Because that page can only insert content into the ha-card

Comment (bramkragten) [reply src/entrypoints/my-redirect.ts:77]: 🤦

**Review by bramkragten — APPROVED** [review]

File: src/entrypoints/my-redirect.ts (hunk 1 above)
Comment (balloob) [reply src/entrypoints/my-redirect.ts:77]: There is probably some eleventy magic I'm missing :)

**Pull request merged**